docs/examples/subset_result.owl

```diff
@@ -25,29 +25,31 @@
     <!-- http://www.geneontology.org/formats/oboInOwl#hasDbXref -->
 
     <owl:AnnotationProperty rdf:about="http://www.geneontology.org/formats/oboInOwl#hasDbXref">
-        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">database_cross_reference</rdfs:label>
+        <rdfs:label>database_cross_reference</rdfs:label>
     </owl:AnnotationProperty>
     
 
 
     <!-- http://www.geneontology.org/formats/oboInOwl#id -->
 
-    <owl:AnnotationProperty rdf:about="http://www.geneontology.org/formats/oboInOwl#id"/>
+    <owl:AnnotationProperty rdf:about="http://www.geneontology.org/formats/oboInOwl#id">
+        <rdfs:label>id</rdfs:label>
```

Comment:
The necessity to add this was the only thing that was really important. @balhoff I wonder if you have an hypothesis why suddenly the id was necessary here to be added?

Reply:
or in other words, why suddenly the OWLAPI decided that in the presence of the 

```
    <owl:AnnotationProperty rdf:about="http://www.geneontology.org/formats/oboInOwl#id"/>
```

Now the `<rdfs:label>id</rdfs:label>` was serialised out?

Reply:
Excellent, thank you for the clarification.